shell/edit/namespace.vue

```diff
@@ -115,6 +118,9 @@ export default {
     },
 
     projectName(newProjectName) {
+      if (newProjectName === null) {
```

Comment:
Projects set as `null` or undefined will automatically turn into `(None)` option, so this is not necessary.

Reply:
If I remove this. It shows an empty option on selection on None.

https://user-images.githubusercontent.com/18264463/225609011-e92f734f-a90a-4c63-92ee-a3c37dc49694.mov

Reply:
No, you need to set a value different from `null`, empty string or undefined.
There's some logic in this dropdown which does not let you do it.
We should not submit the label as value.